tmt/steps/__init__.py

```diff
@@ -392,12 +404,145 @@ def go(self):
         self.verbose('order', self.order, 'magenta', level=3)
 
 
-class Login(tmt.utils.Common):
-    """ Log into the guest """
-
+class ActionStep(tmt.utils.Common):
```

Comment:
A docstring would be nice here. For example `A special step which performs an action during a normal step` or something along those lines.

Reply:
Make sense. Added.